CMAKE.md

```diff
@@ -52,10 +57,7 @@ latest version:
     git pull
     git submodule update --init
```

Comment:
Isn't `git submodule  update --init` rather out of date? Instead: `git pull --recurse=yes`? (The `git clone` has the recursive option)

Reply:
Good point, yes the recurse option has been added to most git commands, including `pull`. 

Presumably it is `--recurse-submodules=yes` or simply `--recurse-submodules`, but seems `--recurse`  also works!